src/Proto3/Suite/DotProto/Internal.hs

```diff
@@ -7,9 +7,9 @@
 {-# LANGUAGE MultiWayIf          #-}
 {-# LANGUAGE NamedFieldPuns      #-}
 {-# LANGUAGE OverloadedStrings   #-}
-{-# LANGUAGE QuasiQuotes         #-}
 {-# LANGUAGE RankNTypes          #-}
 {-# LANGUAGE RecordWildCards     #-}
+{-# LANGUAGE QuasiQuotes         #-}
```

Comment:
Why move this pragma out of its alphabetical position?

Reply:
I removed it and unremoved it.